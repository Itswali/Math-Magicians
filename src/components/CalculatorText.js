import React from 'react';

export default function CalculatorText() {
  return (
    <div>
      <h2 style={{
        fontSize: '3.5rem',
        fontWeight: '700',
        padding: '100px',
        lineHeight: '1.5em',
      }}
      >
        Lets Do Some Math!
      </h2>
    </div>
  );
}
