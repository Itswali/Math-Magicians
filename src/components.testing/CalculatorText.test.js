import React from 'react';
import renderer from 'react-test-renderer';
import CalculatorText from '../components/CalculatorText';

test('Home SnapShot test ', () => {
  const component = renderer.create(
    <CalculatorText />,
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
