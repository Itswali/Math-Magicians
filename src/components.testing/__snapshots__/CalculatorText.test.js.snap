// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home SnapShot test  1`] = `
<div>
  <h2
    style={
      Object {
        "fontSize": "3.5rem",
        "fontWeight": "700",
        "lineHeight": "1.5em",
        "padding": "100px",
      }
    }
  >
    Lets Do Some Math!
  </h2>
</div>
`;
